### PR TITLE
CTI-Mode Muting

### PR DESCRIPTION
**Pull request by bogue**

### CODE REVIEW

File: src/api/ctid-ng.js [hunk 1]
```diff
@@ -115,4 +115,12 @@ export default (client: ApiRequester, baseUrl: string) => ({
 
     return client.post(`${baseUrl}/users/me/faxes?${params}`, fax, headers);
   },
+
+  mute: (callId: string) => client.put(`${baseUrl}/users/me/calls/${callId}/mute/start`, null)
+    .then(() => true)
```
Comment (manuquentin) [inline src/api/ctid-ng.js]: Can you use `ApiRequester.successResponseParser` as last argument ?

File: src/service/CallApi.js [hunk 2]
```diff
@@ -49,4 +49,12 @@ export default class CallApi {
   static async relocateCall(callId: string, line: number, contactIdentifier?: string): Promise<Relocation> {
     return getApiClient().ctidNg.relocateCall(callId, 'line', line, contactIdentifier);
   }
+
+  static async mute(callId: string): Promise<void> {
+    return getApiClient().ctidNg.mute(callId);
+  }
+
+  static async unmute(callId: string): Promise<void> {
+    return getApiClient().ctidNg.unmute(callId);
```
Comment (manuquentin) [inline src/service/CallApi.js]: can you check with @sboily but I think that ctidNg should be replaced by calld.

File: src/utils/api-requester.js [hunk 3]
```diff
@@ -56,7 +56,12 @@ export default class ApiRequester {
   }
 
   static defaultParser(response: Object) {
-    return response.json().then((data: Object) => data);
+    try {
+      return response.json().then((data: Object) => data);
+    } catch (e) {
+      console.warn(e, response);
```
Comment (manuquentin) [inline src/utils/api-requester.js]: Is it for debug purpose ?

Comment (bogue) [reply src/utils/api-requester.js]: oops! it was

File: src/api/ctid-ng.js (hunk 1 above)
Comment (bogue) [reply src/api/ctid-ng.js]: Sorry, that was more debugging.

File: src/service/CallApi.js (hunk 2 above)
Comment (bogue) [reply src/service/CallApi.js]: This has implications larger than the scope of this PR, mind if we "shovel this one forward" to paraphrase an old Québecois saying?

Comment (sboily) [reply src/service/CallApi.js]: Yes we need to use calld instead ctidNg. I have duplicated ctidNg with calld to permit this migration.

Comment (sboily) [reply src/service/CallApi.js]: Yes we need to use calld.

**Pull request merged**